src/components/superAdminHeader/index.tsx

```diff
@@ -0,0 +1,28 @@
+import DownArrowIcon from 'assets/icons/iconDownArrow';
+import { Container } from 'pages/superAdminLayout/styles';
+
+import { AdminAvatar, AdminName, AdminProfile, Header } from './styles';
+
+const name = 'Anna Asol';
+const title = 'Admin';
+const initials = 'AA';
```

Comment:
are those consts needed? or we can get info from db?

Reply:
yes it is needed for now, it will be changed later